core/mocks/generated/trezorio.pyi

```diff
@@ -1,6 +1,116 @@
 from typing import *
 
 
+# extmod/modtrezorio/modtrezorio-fatfs.h
+class FatFSFile:
+    """
+    Class encapsulating file
+    """
+
+    def __exit__(self) -> None:
+        """
+        Close an open file object
+        """
+
+    def close(self) -> None:
+        """
+        Close an open file object
+        """
+
+    def read(self, data: bytearray) -> int:
+        """
+        Read data from the file
+        """
+
+    def write(self, data: bytearray) -> int:
+        """
+        Write data to the file
+        """
+
+    def seek(self, offset: int) -> None:
+        """
+        Move file pointer of the file object
+        """
+
+    def truncate(self) -> None:
+        """
+        Truncate the file
+        """
+
+    def sync(self) -> None:
+        """
+        Flush cached data of the writing file
+        """
+
+
+# extmod/modtrezorio/modtrezorio-fatfs.h
+class FatFSDir:
+    """
+    Class encapsulating directory
+    """
+
+    def __next__(self) -> Tuple[str, int, str]:
+        """
+        Read an entry in the directory
+        """
+
+
+# extmod/modtrezorio/modtrezorio-fatfs.h
+class FatFS:
+    """
+    Class encapsulating FAT filesystem
+    """
+
+    def __init__(self) -> None:
+        """
+        """
+
+    def open(self, path: string, flags: string) -> FatFSFile:
+        """
+        Open or create a file
+        """
+
+    def listdir(self, path: string) -> FatFSDir:
+        """
+        List a directory (return generator)
+        """
+
+    def mkdir(self, path: string) -> None:
+        """
+        Create a sub directory
+        """
+
+    def unlink(self, path: string) -> None:
```

Comment:
type hint: `def unlink(self, path: str) -> None`

Reply:
Fixed in d02c9dc02580a218f5ad5d05c2d058e9790cc255